templates/user/edit-profile.html

```diff
@@ -148,7 +148,12 @@
                                 <td colspan="2"><span class="inline-header">
                                     {% if profile.is_totp_enabled %}
                                         {{ _('Two Factor Authentication is enabled.') }}
-                                        <a href="{{ url('disable_2fa') }}" class="button inline-button">Disable</a>
+					{% if request.user.is_superuser %}
+					    <a href="{{ url('disable_2fa') }}" class="button inline-button">Disable</a>
+					{% else %}
+					    <a href="javascript:alert('{{ _('Contact an admin to disable 2fa')|escapejs }}')"
+					       title="{{ _('Contact an admin to disable 2fa') }}" class="button inline-button">Disable</a>	
```

Comment:
`2fa` -> `Two Factor Authentication`, to stay consistent with the other messages.